src/components/CardContainer/CardContainer.js

```diff
@@ -30,12 +30,26 @@ class CardContainer extends React.Component {
             .then(res => {
                 let random = UniqueRandomArray(res)
                 let dogs = [random(), random()]
-                const state = {...this.state, dogs: dogs}
+                const state = {...this.state, dogs: dogs, randomDog: random}
                 this.setState(state)
                 }
             )
     }
 
+    clickUpdateWin = () => {
+        //Matt work here
+        // document.getElementsByClassName("like-button").addEventListener("click", );
+        let dogs = [this.state.randomDog(), this.state.randomDog()]
+        while ((dogs[0]._id == this.state.dogs[0]._id || dogs[0]._id == this.state.dogs[1]._id)
+            && (dogs[1]._id == this.state.dogs[0]._id || dogs[1]._id == this.state.dogs[1]._id)) {
+            dogs = [this.state.randomDog(), this.state.randomDog()]
+        }
+        const state = {...this.state, dogs: dogs}
+        this.setState(state)
```

Comment:
Maybe pull this out into its own function.

Reply:
This doesn't block merge for me - could equally be done as a cleanup task, & it depends on how difficult to read the other code that's going in this function will be.